src/jit/hwintrinsicxarch.cpp

```diff
@@ -224,6 +224,54 @@ int Compiler::numArgsOfHWIntrinsic(NamedIntrinsic intrinsic)
     return hwIntrinsicInfoArray[intrinsic - NI_HW_INTRINSIC_START - 1].numArgs;
 }
 
+//------------------------------------------------------------------------
+// numArgsOfHWIntrinsic: get the number of arguments based on baseType
```

Comment:
The function header description and arguments need to be updated.

Reply:
Right, missed this one